# Share-Shuffle — a quantitative teardown 🔬
### The net-issuance long-short · the inverted sign · the small-cap / survivorship explanation

![Signal: Weak](https://img.shields.io/badge/Signal-Weak-dab617?style=flat-square)
![Tradability: Mirage](https://img.shields.io/badge/Tradability-Mirage-c0392b?style=flat-square)
![Issuance anomaly on large caps?: Not supported](https://img.shields.io/badge/Issuance_anomaly_on_large_caps%3F-Not_supported-8b949e?style=flat-square)

The deep companion to the [notebook for the curious](01_for_the_curious.ipynb). A real academic anomaly that runs backwards on the tradable end of the market.

> ⚠️ **Not investment advice.** SEC EDGAR diluted shares + Yahoo annual returns, ~412 current S&P 500 members, XBRL era. Sources in [`docs/references.md`](../docs/references.md).

In [1]:
import sys, os
sys.path.insert(0, os.path.abspath("../../.."))  # repo root (quantlab/)
sys.path.insert(0, os.path.abspath(".."))        # study package (share_shuffle/)
%matplotlib inline
import matplotlib.pyplot as plt
plt.rcParams["figure.figsize"] = (10, 5.5); plt.rcParams["axes.grid"] = True
import numpy as np, pandas as pd
from share_shuffle import data, strategy as st
sig, fwd = data.fetch_panel()                  # cache-first (shared EDGAR pull)
h = st.quantile_hedge(sig, fwd, q=0.2, long_high=False)   # long low-issuance / buyback


## Verdict, up front

| Axis | Stamp | Why |
|---|---|---|
| Signal | **Weak** | hedge −2.8%/yr, Sharpe −0.30, t −1.3 — inverted |
| Tradability | **Mirage** | issuers beat buyers on large caps |
| Anomaly on large caps? | **Not supported** | the real effect is a small-cap one |

> 💡 *In plain words:* a genuine factor, measured in the wrong universe.

## 1 · The claim, steelmanned

- **H₁:** high net issuance predicts low forward returns.
- **H₂:** the long-buyback / short-issuer hedge is positive and significant.
- **H₃:** it's tradable on liquid large caps (not just micro-caps).

## 2 · So what? — what rides on each

H₁/H₂ would make issuance a free, fundamentals-based signal straight from 10-Ks. H₃ decides whether it survives where you can actually trade. A small-cap-only effect is real science but useless to a large-cap book.

## 3 · How we'd know — the protocol

EDGAR diluted shares → net issuance (YoY change) → annual quintile sort → long bottom-20% (buyers), short top-20% (issuers) → next-year hedge, Sharpe, t-stat, hit rate over the XBRL window.

## 4 · The teardown

### 4.1 The long-short, year by year

In [2]:
disp=h.copy(); disp.columns=['issuer leg','buyer leg','hedge (buyer-issuer)']
display((disp*100).round(1))
s=st.summary(h['hedge']); print({k:round(v,3) for k,v in s.items()})

,issuer leg,buyer leg,hedge (buyer-issuer)
2008,36.6,34.7,-1.9
2009,25.3,28.2,3.0
2010,-1.3,8.1,9.5
2011,26.5,24.0,-2.6
2012,39.5,48.2,8.8
2013,21.6,21.1,-0.5
2014,12.0,6.7,-5.3
2015,20.8,20.3,-0.6
2016,27.0,26.5,-0.6
2017,-4.3,-6.0,-1.7


{'mean': -0.028, 'vol': 0.093, 'sharpe': -0.303, 'tstat': -1.287, 'hit_rate': 0.278, 'max_drawdown': -0.533, 'n': 18}


> 💡 *In plain words:* the hedge mean is **negative** (−2.8%/yr) and wins only ~28% of years. **H₁ and H₂ are rejected on this universe** — the sign is backwards.

### 4.2 Why the sign inverts — H₃ fails

Three forces flip it on large caps:

1. **Issuers were the winners.** 2010–2025's heavy large-cap issuers were high-growth tech diluting via stock-based comp and M&A — the market leaders.
2. **The anomaly lives in small caps.** Pontiff-Woodgate's effect concentrates in small, illiquid names; Hou-Xue-Zhang (2020) show financing anomalies fade with value-weighting and NYSE breakpoints.
3. **Survivorship.** Current members exclude the dilutive blow-ups that delisted — which would have helped the short — yet even this biased test can't find the effect. **H₃ rejected.**

## 5 · The verdict

H₁/H₂/H₃ all rejected on tradable large caps → Signal `WEAK` (inverted), Tradability `MIRAGE`, anomaly-on-large-caps `NOT SUPPORTED`. The fingerprinted run is in [`docs/results.md`](../docs/results.md).

## 6 · Could you trade it?

Not on this universe. To harvest the real premium you'd need a small-cap, value-weighted, NYSE-breakpoint book with the delisted names included — outside what a free EDGAR pull on current S&P members can offer, and outside what most large books can size.

## 7 · Going further

Forks: (a) a Russell 2000 / micro-cap universe; (b) composite issuance (Daniel-Titman) vs raw share count; (c) combine with accruals ([52](../../52-smoke-screen/)) and asset growth ([44](../../44-growth-spurt/)) into a financing factor. Backlog: [`docs/pwb_strategies_inventory.md`](../../../docs/pwb_strategies_inventory.md).